contractgen/src/test/java/org/apache/servicecomb/toolkit/contractgen/DefaultContractsGeneratorTest.java

```diff
@@ -85,7 +86,11 @@ public void testGenerate()
 
     DefaultContractsGenerator defaultContractsGenerator = new DefaultContractsGenerator();
     defaultContractsGenerator.configure(config);
-    assertTrue(defaultContractsGenerator.generate());
+    try {
+      defaultContractsGenerator.generate();
+    } catch (RuntimeException e) {
+      fail();
```

Comment:
It's better to add some information which case failure in the fail method. 
It's could save us lot of time to find out the CI build error.